packages/core/useScroll/index.ts

```diff
@@ -109,12 +109,18 @@ export function useScroll(
       e.target === document ? (e.target as Document).documentElement : e.target
     ) as HTMLElement
 
+    // We have to check if the scroll amount is close enough to some threshold in order to
+    // more accurately calculate arrivedState. This is because scrollTop/scrollLeft are non-rounded
+    // numbers, while scrollHeight/scrollWidth and clientHeight/clientWidth are rounded.
+    // https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollHeight#determine_if_an_element_has_been_totally_scrolled
+    const arrivedStateThresholdPixels = 1
```

Comment:
A bit nitpicking but can we move it to top-level with UPPER_CASE? Thanks.

Reply:
How does this look? Thanks! https://github.com/vueuse/vueuse/commit/a0d28a784659647a19c68eaedb6d0241d5cad4f9